example-app/database/migrations/2024_05_28_195716_create_flights_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('flights', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('airline_id');
+            $table->foreignId('departure_city_id');
+            $table->foreignId('arrival_city_id');
+            $table->timestamp('departure_time');
```

Comment:
should be `departure_time`? or `departure_datetime`?

Reply:
I think it would be better to declare it as datetime

https://laravel.com/docs/11.x/migrations#column-method-dateTime
https://laraveldaily.com/post/laravel-datetime-vs-timestamp-differences